src/renderer/vt/vtrenderer.hpp

```diff
@@ -195,6 +195,8 @@ namespace Microsoft::Console::Render
         [[nodiscard]] HRESULT _SetInvisible(const bool isInvisible) noexcept;
         [[nodiscard]] HRESULT _SetCrossedOut(const bool isCrossedOut) noexcept;
         [[nodiscard]] HRESULT _SetReverseVideo(const bool isReversed) noexcept;
+        [[nodiscard]] HRESULT _SetHyperlink(const std::wstring& uri, const std::wstring& customId) noexcept;

```

Comment:
nit: prefer string views for this

Reply:
(makes the API more flexible -- caller can send anything that looks like a set of characters at runtime, instead of just "always a wstring")